src/hip/hip_build_utils.cpp

```diff
@@ -39,6 +39,72 @@
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_VERBOSE)
 MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_DEBUG_HIP_DUMP)
 
+#if MIOPEN_OFFLINE_COMPILER_PATHS_V2
+
+// Include rocm-core header for get ROCm Install Path Method
+#include <rocm-core/rocm_getpath.h>
+
+// Flags to hold Relative Directory Path
+// for each Compiler Flags from ROCM Install Path
+// This flag Paths are expected to be deprecated/modified
+// in upcoming MAJOR Releases.
+#define MIOPEN_CLANG_REL_PATH "llvm/bin/clang"
+#define MIOPEN_OCL_REL_PATH "bin/clang-ocl"
```

Comment:
@arvindcheru `clang-ocl` has been deprecated so we need to find an alternative from the compiler folks.

Reply:
@junliume clang-ocl is not deprecated yet. Its just removed from rocm-dev meta package. 
In ci as discussed offline, rocm-clang-ocl can be installed separately.